docs/core/whats-new/index.md

```diff
@@ -3,15 +3,17 @@ title: What's new in .NET Core
 description: Learn about the new features found in .NET Core.
 author: rpetrusha
 ms.author: ronpet
-ms.date: 05/07/2018
+ms.date: 12/4/2018
```

Comment:
format is mm/dd/yyyy. cats will complain if the two digits are not there later 😄